README.md

```diff
@@ -34,37 +34,39 @@
 
 ## WORK IN PROGRESS
 
-Please note that this plugin is under active development, and might not be ready to run on production code yet.
-
-## Contributing
-
-If you're interested in contributing to the development of Prettier for PHP, you can follow the [CONTRIBUTING guide from Prettier](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md), as it all applies to this repository too.
-
-To test it out on a PHP file:
-
-* Clone this repository.
-* Run `yarn`.
-* Create a file called `test.php`.
-* Run `yarn prettier test.php` to check the output.
+Please note that this plugin is currently in alpha stage and still under active development. We encourage everyone to try it and give feedback, but we don't recommend it for production use yet.
 
 ## Install
 
 yarn:
+
 ```bash
 yarn add --dev prettier/prettier prettier/plugin-php
```

Comment:
Or should we prefer global installation?

Reply:
@czosel Let's do this as do `prettier` here https://prettier.io/docs/en/install.html